### PR TITLE
Auditor is an example of a Relying Party.

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
+: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service. An Auditor is an example of a Relying Party.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:198]:
```suggestion
: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
An Auditor is an example of a Relying Party.
```

**Comment by rjb4standards** [comment]
@SteveLasker I'm concerned that this "overloading" of Auditor semantics is going to create ambiguities and confusion.
I continue to believe we need to define two distinctive roles; A Gatekeeper that ensures only verified "signed statements" following registration policies are placed into the registry and a "Relying Party" that "trusts" the information contained in a Registry.
IMO, an auditor is the "checker" making sure that policies are being followed, which adds integrity to the Registry processes and procedures and the Registry operator (Transparency Service) is properly following the SCITT protocol and registration policy processes and procedures.

**Comment by roywill** [comment]
I am fine with this change.  The larger issue that Steve points out I believe can be left out.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by roywill — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -196,6 +196,7 @@ Artifact:
 Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
+An Auditor is an example of a Relying Party.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:199]:
```suggestion
An Auditor is an example of a specialized Relying Party.
```

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
not only an example, but also literally a specialization (narrower term)

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**